### ISSUE TITLE
[ui5-multi-input]: Incorrect screen reader announcement after selecting suggestion item

### ISSUE THREAD

**Issue by zhangliangyi** [UI5/webcomponents#12097]
### Bug Description

After selecting some item in the suggestion picker from a multi-input component, the screen reader announces "undefined List item 1 of 12"

### Affected Component

ui5-multi-input

### Expected Behaviour

The announcement should be meaningful for the the screen reader.

### Steps to Reproduce

1. Go to https://sap.github.io/ui5-webcomponents/components/MultiInput/#addremove-tokens
2. Type "a" and select "Argentina"
3. Observe the annoucement from screen reader

### Priority

None

### UI5 Web Components Version

2.13.2

### Browser

Chrome

### Organization

SF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

The issue is reproducible with JAWS Version 2025.2506.170:
<img width="1176" height="684" alt="Image" src="https://github.com/user-attachments/assets/bb22911f-9fa9-4fb1-ad27-d6e24b6498ee" />

Could you please take a look at this?

Best,
Siyana
